business-profile-render.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /**
- * Plugin Name: Business Profile Data
- * Plugin URI: /wp-admin/tools.php?page=businessprofiledata
```

Comment:
May want both urls so that user know that the plugin description/how to use guide lives under tools tab?

Reply:
This URL has its dedicated purpose. If you search for `$plugin['PluginURI']` in this PR you would see it is actually used by the plugin information page